docs/index.rst

```diff
@@ -10,16 +10,14 @@ To post questions about anything to do with pytentiostat, please submit an issue
 
 pytentiostat provides a Python API package to the JUAMI potentiostat
 
-The goal of the xpdAn package is to simplify your analysis workflow during beamtime, so that you can focus more on the scientific aspects of your experiment.
-
 To get started, please go to :ref:`quick_start`
 
-If you have suggestions for new features in xpdAcq, or want to report a bug or simply ask a question about the software, please post it as a new thread at XPD-Users
+If you have suggestions for new features in pytentiostat, or want to report a bug or simply ask a question about the software, please post it as a new thread at XPD-Users
```

Comment:
This is referring to a Google group ``XPD-users``.   We may want to start a Google group called ``pytentiostat-users``, or perhaps we could use the juami-exchange google group.   Google groups are good because they act like stackexchange for our products.  People can post questions there that can be answered and then everyone in the community can see the questions and the answers, and can search the group if they run into a problem to see if anyone has asked a question on the same topic any time before.

Reply:
I think this index file was brought in with a different pull request that you might have made. I have not used this index.rst so far in my html construction. Do you intend for this message to be on the front page of the documentation? In that case I can add it to the pytentiostat_documentation file.

Creating a google group for users is a good idea, although the difference between the google group vs creating issues on GitHub isn't clear to me.